tests/testthat/test-order-by-date.r

```diff
@@ -0,0 +1,93 @@
+library(testthat)
+library(healthcareai)
+context("Checking that the orderByDate function is working correctly")
+
+
+df1 <- data.frame(date = c('2009-01-01','2010-01-01','2009-03-08','2009-01-19'),
+                  a = c(1,2,3,4))
+
+
+test_that("We have multiple columns and dates without timestamps and checking if ascending", {
+  df2 <- data.frame(date = c('2009-01-01','2009-01-19','2009-03-08','2010-01-01'),
+             a = c(1,4,3,2))  
+  rownames(df2) <- c(1,4,3,2)
+  df2$date <- lubridate::ymd_hms(df2$date, truncated = 5) # makes sure that df2 date is in POSIXct format
+  df3 <- orderByDate(df1,'date', descending = FALSE)
+  expect_equal(all(df3 == df2),TRUE) # just tests the the values rather than data frame object
+})
+
+
+test_that("We have multiple columns and dates without timestamps and checking if descending", {
+  df2 <- data.frame(date = c('2010-01-01','2009-03-08','2009-01-19','2009-01-01'),
+                    a = c(2,3,4,1))  
+  rownames(df2) <- c(2,3,4,1)
+  df2$date <- lubridate::ymd_hms(df2$date, truncated = 5) # makes sure that df2 date is in POSIXct format
+  df3 <- orderByDate(df1,'date', descending = TRUE)
+  expect_equal(all(df3 == df2),TRUE) # just tests the the values rather than data frame object
+})
+
+
+df1 <- data.frame(date = c('2009-03-08 12:59','2009-01-01 23:59','2010-01-01 23:59','2009-03-08 23:59','2009-01-19 23:59','2010-01-01 12:59'),
+                  a = c(1,2,3,4,5,6))
```

Comment:
Would you mind staying under 80 characters (by indenting)?

RStudio can help: Tools -> Global Options -> Code - > Display - Set Margin column to 80